lib/shopify-cli/commands/system.rb

```diff
@@ -139,6 +140,15 @@ def display_environment
         end
         @ctx.puts("")
       end
+
+      def shopifolk?
+        is_shopifolk = ShopifyCli::Shopifolk.check
+        if is_shopifolk
+          @ctx.puts("{{v}} you are a shopifolk!")
```

Comment:
For formatting, how about a new section, e.g.:

```
Identity (in bold)
  We have identified you as Shopify staff
```
and for non-staff we'll just now show this section.